### PR TITLE
Add send slack massage to a channel

### PR DESCRIPTION
**Pull request by lioramilbaum**
I took the changes from this PR and added functionality
https://github.com/devopsloft/devopsloft/pull/114

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,20 +16,13 @@ doc/build/*
...
-!.elasticbeanstalk/*.global.yml
-
 .vagrant/
 Vagrantfile.local
 aws.yml.local
 devopsloft_workspace.code-workspace
 aws_ubuntu.pem
 *~
 .vscode/
+.secrets.json
```
Comment (ShimiTaNaka) [inline .gitignore]: As discussed in the original PR, this is redundant

File: Vagrantfile [hunk 2]
```diff
@@ -5,6 +5,11 @@
 # Exit if no environment name specified.
 # -------------------------------------------------------------
 
+$script = <<-SCRIPT
+docker cp /vagrant/.secrets.json web:/.secrets.json
+docker exec web ./events.py
```
Comment (ShimiTaNaka) [inline Vagrantfile]: We should probably later convert these jsons to env files and then it's easier to run docker with `--env_file`
we should put it in the lists for the future

File: devopsloft.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (ShimiTaNaka) [inline devopsloft.yml]: I think this should followed up with a README note to explain the meaning of the file

File: docker/events.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    events = client.GetEvents({'group_urlname': 'DevOps-Loft',
+                              'status': status})
+    events = json.dumps(events.__dict__)
+    events = json.loads(events)
+    return events
+
+
+def putPost(service='Slack', channel='sandbox', message='Hello World',
+            event=None):
+    try:
```
Comment (ShimiTaNaka) [inline docker/events.py]: Suggest throwing an error if a value other than 'Slack' is passed or rename the method and remove the variable, if someone will use the function in the future, he will have no indication why it doesn't work

File: .gitignore (hunk 1 above)
Comment (lioramilbaum) [reply .gitignore]: Not sure I agree now. I have such a file and I don't want git tracking it.

File: Vagrantfile (hunk 2 above)
Comment (lioramilbaum) [reply Vagrantfile]: agree. go ahead.

File: devopsloft.yml (hunk 3 above)
Comment (lioramilbaum) [reply devopsloft.yml]: +1

File: docker/events.py (hunk 4 above)
Comment (lioramilbaum) [reply docker/events.py]: +1

File: .gitignore (hunk 1 above)
Comment (ShimiTaNaka) [reply .gitignore]: But there is already the `.secret*` in the gitignore...

Comment (lioramilbaum) [reply .gitignore]: +1

File: devopsloft.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+
+publish: "disabled"
```
Comment (ShimiTaNaka) [inline devopsloft.yml]: Possibly add some comment on the Feature flag, not a must

Comment (lioramilbaum) [reply devopsloft.yml]: I added one in the README.md file

**Pull request merged**